pkg/config/oc/bgp_configs.go

```diff
@@ -3175,9 +3215,6 @@ type NeighborConfig struct {
 	// original -> bgp:peer-group
 	// The peer-group with which this neighbor is associated.
 	PeerGroup string `mapstructure:"peer-group" json:"peer-group,omitempty"`
-	// original -> gobgp:send-software-version
-	// gobgp:send-software-version's original type is boolean.
-	SendSoftwareVersion bool `mapstructure:"send-software-version" json:"send-software-version,omitempty"`
```

Comment:
Honestly, I do not know why this didn't surface up earlier. 
`neighbor-interface` is defined at line 919 https://github.com/osrg/gobgp/blob/master/tools/pyang_plugins/gobgp.yang#L919
`send-software-version` at 1010: https://github.com/osrg/gobgp/blob/master/tools/pyang_plugins/gobgp.yang#L1010
It is logical that it goes after it. 

Changing order of field broke test, though, so I fixed `overrideConfig()` in a separate commit